core/src/main/scala/kafka/log/FileMessageSet.scala

```diff
@@ -241,7 +241,7 @@ class FileMessageSet private[kafka](@volatile var file: File,
       case dc => channel.transferTo(position, count, dc)
     }).toInt
     trace("FileMessageSet " + file.getAbsolutePath + " : bytes transferred : " + bytesTransferred
-      + " bytes requested for transfer : " + math.min(size, sizeInBytes))
+      + " bytes requested for transfer : " + count)
```

Comment:
While you're at it, can you please replace string concatenation with string interpolation (http://docs.scala-lang.org/overviews/core/string-interpolation.html)?